Common/src/main/java/com/tabnineCommon/chat/commandHandlers/InitHandler.kt

```diff
@@ -24,8 +25,12 @@ class InitHandler(gson: Gson) : ChatMessageHandler<Unit, InitPayload>(gson) {
         val colorPalette = readColorPalette()
         val isDarkTheme = EditorColorsManager.getInstance().isDarkEditor
         val font = EditorColorsManager.getInstance().globalScheme.getFont(EditorFontType.PLAIN).size
-
-        return InitPayload("ij", isDarkTheme, colorPalette, font, isTelemetryEnabled(), getServerUrl())
+        val serverUrl = if (Config.IS_SELF_HOSTED) {
+            getServerUrl()
```

Comment:
Why isn't this condition baked into the `getServerUrl` method?

Reply:
no, but I get your comment. will move it